tcl/filter.tcl

```diff
@@ -11,19 +11,22 @@ proc qc::filter_validate {event {error_handler qc::error_handler}} {
     ::try {
         set url_path [qc::conn_path]
         set method [string toupper [qc::conn_method]]
-        # Check if this request is registered for this filter and a handler exists to validate against.
-        if { [qc::filter_handler_exists filter_validate $url_path $method] && [qc::handlers exists $url_path $method] } {
+        # Check if this request is registered and a handler exists to validate against.
+        if { [qc::registered $method $url_path] && [qc::handlers exists $method $url_path] } {
             # Validate to data model
-            qc::handlers validate2model $url_path $method
+            qc::handlers validate2model $method $url_path
             # Custom validation
-            if {[qc::handlers validation exists $url_path $method]} {
-                qc::handlers validation call $url_path $method
+            if {[qc::handlers validation exists $method $url_path]} {
+                qc::handlers validation call $method $url_path
             }
             
             # Let the client know if there's a problem
             if {[qc::conn_open] && $method ne "GET" && ! [qc::record all_valid]} {
```

Comment:
The idea is to only return the JSON response if the method is a type of `POST` request i.e. `DELETE` `RESTORE` `POST` etc.

`HEAD` should be treated like `GET` and excluded. Will update.